sql/core/src/main/scala/org/apache/spark/sql/execution/ui/AllExecutionsPage.scala

```diff
@@ -103,6 +104,9 @@ private[ui] abstract class ExecutionTable(
       <td>
         {descriptionCell(executionUIData)}
       </td>
+      <td>
+        {executionUIData.sqlText.getOrElse("-")}
```

Comment:
Is there a reason to insert a "-" rather than leave this blank in the case it's not available?